ArrayIndexOutOfBoundsException in preLayoutChildren()

It looks like when the tops are copied into the bottom,
there is a mismatch in size between
the array **src** (`mColumnTops`) and the array **dest** (`mColumnBottoms`).

``` java
02-11 14:10:48.628  3518  6271 E AndroidRuntime: Caused by: java.lang.ArrayIndexOutOfBoundsException: src.length=3 srcPos=0 dst.length=0 dstPos=0 length=3
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at java.lang.System.arraycopy(Native Method)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at com.etsy.android.grid.StaggeredGridView.preLayoutChildren(StaggeredGridView.java:341)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at com.etsy.android.grid.StaggeredGridView.layoutChildren(StaggeredGridView.java:326)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at com.etsy.android.grid.ExtendableListView.onLayout(ExtendableListView.java:507)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at com.etsy.android.grid.ExtendableListView.updateEmptyStatus(ExtendableListView.java:2040)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at com.etsy.android.grid.ExtendableListView.access$1000(ExtendableListView.java:59)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at com.etsy.android.grid.ExtendableListView$AdapterDataSetObserver.onChanged(ExtendableListView.java:2078)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at android.database.DataSetObservable.notifyChanged(DataSetObservable.java:37)
02-11 14:10:48.628  3518  6271 E AndroidRuntime:    at android.widget.BaseAdapter.notifyDataSetChanged(BaseAdapter.java:50)
```

--- a/library/src/main/java/com/etsy/android/grid/StaggeredGridView.java
+++ b/library/src/main/java/com/etsy/android/grid/StaggeredGridView.java
@@ -1244,6 +1244,7 @@ public class StaggeredGridView extends ExtendableListView {
         GridListSavedState ss = (GridListSavedState) state;
         mColumnCount = ss.columnCount;
         mColumnTops = ss.columnTops;
+        mColumnBottoms = new int[mColumnCount];
         mPositionData = ss.positionData;
         mNeedSync = true;
         super.onRestoreInstanceState(ss);

